### PR TITLE
Make packetCycleTest run for all versions & seperate lint step from test step

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by rom1504** [comment]
why is it not failing? it should fail until mc packets is updated

**Comment by u9g** [comment]
> why is it not failing? it should fail until mc packets is updated

didn't add env to yml, for now nothing is in env, that way while nothing is in the env var it runs all

**Comment by u9g** [comment]
> why is it not failing? it should fail until mc packets is updated

Okay, failing now :)

**Comment by u9g** [comment]
Not understanding why it's just not running the test now?

File: test/packetCycleTests.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const serializers = {}
+const deserializers = {}
+const makeClientSerializer = version => createSerializer({ state: states.PLAY, version, isServer: true })
+const makeClientDeserializer = version => createDeserializer({ state: states.PLAY, version })
+
+describe('Packet cycle tests', () => {
+  for (const supportedVersion of supportedVersions) {
+    const mcData = require('minecraft-data')(supportedVersion)
+    const version = mcData.version.minecraftVersion
+    it('Packet cycle tests for ${version}', () => {
```
Comment (rom1504) [inline test/packetCycleTests.js:16]: Is that the right thing

Comment (u9g) [reply test/packetCycleTests.js]: fixed, but now the grep isn't being called: https://github.com/PrismarineJS/node-minecraft-protocol/runs/4488855401?check_suite_focus=true#step:6:13

**Comment by rom1504** [comment]
* [ ] use the nearest major available data ; or remap/redump in minecraft packets
* [ ] dump 1.17 and 1.18

**Comment by rom1504** [comment]
stuck pr

**Comment by rom1504** [comment]
would be nice to finish

**Comment by rom1504** [comment]
please re-open if you want to finish it

**Pull request closed without merging**